docs/src/examples/components/Popup/BestPractices/PopupBestPractices.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
+    content: 'Use popupFocusTrapBehavior if the focus needs to be trapped inside of the Popup.',
```

Comment:
What does `trapped` mean here - just that the focus remains inside the popup until the popup is closed? What is the default behavior if you don't have this set?

Reply:
Trapped means that as long as the popup is opened, the focus stays inside of it. But we will add more details about it as part of the issue #1572